src/ResourceManager/ServiceBus/Commands.ServiceBus/Models/ListKeysAttributes.cs

```diff
@@ -35,6 +35,8 @@ public ListKeysAttributes(AccessKeys listKeysResource)
                 PrimaryKey = listKeysResource.PrimaryKey;
                 SecondaryKey = listKeysResource.SecondaryKey;
                 KeyName = listKeysResource.KeyName;
+				AliasPrimaryConnectionString = listKeysResource.AliasPrimaryConnectionString;
```

Comment:
@v-Ajnava nit: indentation is off a little for this line

Reply:
Corrected